### ISSUE TITLE
Use AS3 to put a datagroup as HTTP URI condition in a Local Traffic Policy

### ISSUE THREAD

**Issue by renatonader** [F5Networks/f5-appsvcs-extension#471]
Trying to use AS3 to configure a Local Traffic Policy that uses a datagroup do match an HTTP URI condition, but couldn't find a way to declare it.

Have already tried to use "datagroup": { "bigip": "/Common/images" } as a value or like the following. All tries return with 422 errors.
```
 "conditions": [
                            {
                                "event": "request",
                                "index": 1,
                                "name": "1",
                                "path": {
                                    "operand": "matches",
                                    "datagroup": { "bigip": "/Common/images" }                          
                                },
                                "type": "httpUri"
                            }
                        ],
```
Is it possible to do that? How to declare?

Thanks!

**Comment by renatonader** [comment]
Hello guys.. 

Any update about this?

I've also open a case #C3615034 for this request.

Thanks.

**Comment by dstokesf5** [comment]
Thank you for your feedback. I have added this request to our internal product backlog as AUTOTOOL-2728.

**Comment by dstokesf5** [comment]
This has been resolved in the 3.32.0 release.
